contrib/cirrus/win-sess-launch.ps1

```diff
@@ -2,24 +2,74 @@
 # tunnels the output such that WSL operations will complete
 $ErrorActionPreference = 'Stop'
 
+$syms =  [char[]]([char]'a'..[char]'z') +
+        [char[]]([char]'A'..[char]'Z') +
+        [char[]]([char]'0'..[char]'9')
+
 if ($Args.Length -lt 1) {
     Write-Object "Usage: " + $MyInvocation.MyCommand.Name + " <script>"
     Exit 1;
 }
 
-function RegenPassword {
-    param($username)
-    $syms = [char[]]([char]'a'..[char]'z' `
-               + [char]'A'..[char]'Z'  `
-               + [char]'0'..[char]'9')
+
+# Shuffle a 32 character password from $syms
+function GenerateRandomPassword {
+    param(
+        [int]$length = 32,
```

Comment:
the length argument doesn't seem to be used in this function, I suppose it should be used in the `$rnd = [byte[]]::new(32)` call and not hard coding 32 there?

Reply:
missed it in reformatting my code, ty